lib/wiki_course_output.rb

```diff
@@ -46,7 +48,7 @@ def course_details_and_description
   end
 
   def course_details
-    # TODO: add support for multiple instructors, multiple content experts.
+    # TODO: add support for multiple content experts.
```

Comment:
Can you include an example in a comment of what the output of this template looks like with multiple instructors?

Reply:
Okay